ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-hosted-pipeline.yml

```
@@ -51,12 +51,17 @@ spec:
     - name: hydra_secret_name
       description: Kubernetes secret name that contains the Hydra credentials.
       default: hydra-credentials
-    - name: hydra_secret_username_key
-      description: The key within the Kubernetes secret that contains the Hydra username.
-      default: username
-    - name: hydra_secret_password_key
-      description: The key within the Kubernetes secret that contains the Hydra password.
-      default: password
+    - name: hydra_secret_sso_client_id_key
+      description: |
+        The key within the Kubernetes secret that contains Hydra SSO client ID.
+      default: sso_client_id
+    - name: hydra_secret_sso_client_secret_key
+      description: |
+        The key within the Kubernetes secret that contains the Hydra SSO password.
+      default: sso_client_secret
+    - name: hydra_secret_sso_token_url_key
+      description: The key within the Kubernetes secret that contains SSO URL.
+      default: sso_token_url
```

```suggestion
    - name: hydra_secret_sso_token_url
      description: The Red Hat SSO URL.
      default: https://auth.redhat.com/auth/realms/EmployeeIDP/protocol/openid-connect/token
```
If the sso_token_url is not saved within a secret (like I suggested above), then this will need to be changed as well, and tasks that use this var will need to be adjusted accordingly.

---

Also if you do the change you need to change also pipeline triggers as they pass non-default parameters.